packages/node/src/index.ts

```diff
@@ -475,10 +476,12 @@ export const build: BuildV3 = async ({
     const experimentalResponseStreaming =
       staticConfig?.experimentalResponseStreaming === true ? true : undefined;
 
+    const { runtime } = nodeVersion;
```

Comment:
We already have `nodeVersion.major` so I think we should use that instead to pass into `detectServerlessLauncherType()`

Reply:
Thanks! I'll pass `detectServerlessLauncherType` the `runtime.nodeVersion` since it's available.
I need this function because it's used in two places: here and in the dev server.